amazonmusic.py

```diff
@@ -458,7 +759,7 @@ class Station:
     * `tracks` - Iterable generator for the `Tracks` that make up this station.
     """
 
-    def __init__(self, am, asin, data):
+    def __init__(self, am, data):
```

Comment:
`create_station()` now has an error due to this change. Will need to remove the now extraneous argument from line 223.

https://github.com/Jaffa/amazon-music/blob/fdb1625140f7e95bfc440a72dfec25eecfb14352/amazonmusic.py#L223